src/fmi4c.c

```diff
@@ -1690,27 +1714,24 @@ bool loadFunctionsFmi2(fmiHandle *fmu, fmi2Type fmuType)
 #endif
 
     char dllPath[FILENAME_MAX] = {0};
-    strncat(dllPath, fmu->unzippedLocation, sizeof(dllPath)-strlen(dllPath)-1);
-#if defined(_WIN32)
-    strncat(dllPath, "\\binaries\\win64\\", sizeof(dllPath)-strlen(dllPath)-1);
-#elif defined(__CYGWIN__)
-    strncat(dllPath, "/binaries/win64/", sizeof(dllPath)-strlen(dllPath)-1);
-#else
-    strncat(dllPath, "/binaries/linux64/", sizeof(dllPath)-strlen(dllPath)-1);
-#endif
+    size_t max_num_chars_to_append = sizeof(dllPath)-1;
```

Comment:
Maybe use camel case instead. I have tried to use camel case for the actual variable names and underscore for identifiers and prefixes. It is not consistent throuhout the entire project anyway though, so not super important.